app/src/main/java/edu/uc/reedws/musiclink/MainActivity.kt

```diff
@@ -56,18 +60,31 @@ class MainActivity() : AppCompatActivity() {
 
     // Method to show Dialog Screen for adding a new playlist
     private fun showAlertDialog(viewModel: ApplicationViewModel) {
-
         val addPlaylistDialogBuilder = AlertDialog.Builder(this)
         val inflater = layoutInflater
         addPlaylistDialogBuilder.setTitle("Enter name of playlist to create")
         val dialogLayout = inflater.inflate(R.layout.add_playlist_dialog, null)
-        val editText  = dialogLayout.findViewById<EditText>(R.id.editText)
+        val searchedSong  = dialogLayout.findViewById<EditText>(R.id.searchedSong)
+
```

Comment:
Gave variable a meaningful name. editText doesn't say anything that it is, so I added switched the name from editText to searchedSong because that user search song using this text field.

Reply:
This is actually not the searched song, but the name of the playlist the user wants to create.
...and as I write this, I realize the point you are making haha

Reply:
This is actually not the searched song, but the name of the playlist the user wants to create.
...and as I write this, I realize the point you are trying to make haha

Reply:
It's fine, I thought that the user can search for the song in the search bar and that is why I named the variable `searchedSong` that stores searched songs by a user.